### DISCUSSION TITLE
Permissionproblem for default user?

### DISCUSSION THREAD

**Discussion by simpsonetti** [Ylianst/MeshCentral#5876] [Q&A] [answered]
Hi, 
in addition to this Discussion from me https://github.com/Ylianst/MeshCentral/discussions/5784 were facing a poc for our production server. I will all users add manualy without any seperate directory to auth. 
So if i will add some users by importing or by manually adding, i have to be sure, that all users couldn't add new devices and new devicegroups by default. 
This is very important. To is there any chance of editing any file to change this behavior?
Regards Sebastian

**Comment by simpsonetti** [comment]
Is there a doc or wiki site, where these rights are shown so i could use them?
Something like:
noNewDeviceGroups, noNewDevices, 
and so on. Maybe somewhere in sourcecode?

**Reply by si458** [reply, marked as answer]
sadly not, the new docs site is here https://ylianst.github.io/MeshCentral/ but it still needs updating!
source code for the doc site is here if you wanted to add to it for us and submit a PR!
https://github.com/Ylianst/MeshCentral/tree/master/docs/docs
